### PR TITLE
catch: Update to version 1.3.1.

### PR DESCRIPTION
**Pull request by DavidEGrayson**
For the packages I contribute, I try to monitor the upstream projects for new releases.  A new version of Catch was released just yesterday, so this pull request updates our MSYS2 package.

### CODE REVIEW

**Comment by mingwandroid** [comment]
Thanks!

File: mingw-w64-catch/PKGBUILD [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 _realname=catch
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
```
Comment (Alexpux) [inline mingw-w64-catch/PKGBUILD]: Add please "pkgbase" entry before "pkgname"

**Comment by DavidEGrayson** [comment]
OK, I did that.  I also reset the pkgrel to 1, which I should have done originally I think.

**Comment by DavidEGrayson** [comment]
Thanks, @Alexpux!  :christmas_tree:

**Pull request merged**